erpnext_documentation/www/docs/user/manual/en/human-resources/shift-management.md

```diff
@@ -16,21 +16,21 @@ The Shift Type document allows you to define the different types of Shifts in yo
 To access Shift Type, go to:
 > Home > Human Resources > Shift Management > Shift Type
 
```

Comment:
Add pre-requisites section

Reply:
Shift Type is itself master there are no prerequisites, but added in shift request.